testing-modules/selenium-junit-testng/src/test/java/com/baeldung/selenium/webdriver/SeleniumWebDriverUnitTest.java

```diff
@@ -0,0 +1,46 @@
+package com.baeldung.selenium.webdriver;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+public class SeleniumWebDriverUnitTest {
+
+    private WebDriver driver;
+
+    private static final String URL = "https://duckduckgo.com/";
+    private static final String INPUT_ID = "search_form_input_homepage";
+
+    @BeforeEach
+    public void setUp() {
+
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+
```

Comment:
remove the empty lines as per guidelines (no blank lines before/after curly braces)

Reply:
thanks, @KevinGilmore 
Fixed